probe/probe.go

```diff
@@ -436,34 +442,35 @@ func getKubeArmorContainers(c *k8s.Client, o Options) {
 
 }
 
-func probeRunningKubeArmorNodes(c *k8s.Client, o Options) error {
+func probeRunningKubeArmorNodes(c *k8s.Client, o Options) (map[string]string, error) {
 
 	// // KubeArmor Nodes
+	postureData := make(map[string]string)
 	nodes, err := c.K8sClientset.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
 	if err != nil {
 		log.Println("error getting nodes", err)
-		return err
+		return postureData, err
```

Comment:
Best practice - Prefer returning nil instead of empty maps. [Ref](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)
```suggestion
		return nil, err
```